sphinx/themes/sphinx13/layout.html

```diff
@@ -1,8 +1,8 @@
 {#
-    sphinxdoc/layout.html
+    sphinx13/layout.html
```

Comment:
I feel strange for the name "sphinx13". It should be renamed if we make it public. But we've already used "sphinxdoc"...

Reply:
I don't care about the name.
I was just trying to make a minimal change and fix obvious errors.
If you like to change the name, feel free to do so.
If you like me to change the name, please tell me the name you want to have.